pkg/graphql/spawn.go

```diff
@@ -211,8 +211,9 @@ func (spawner *Spawner) applyVolumeForGroup(launchGroup string, group DtoGroup)
 		return
 	}
 
-	name := "project-" + strings.ToLower(group.Name)
-	mountPath := "/project/" + group.Name
+	groupName := strings.ToLower(strings.ReplaceAll(group.Name, "_", "-"))
```

Comment:
Not sure about it but this looks a little bit like `EscapeToDNSLabel ` here.

https://github.com/InfuseAI/primehub-controller/blob/master/pkg/escapism/escapism.go#L25

Reply:
Yes, but now follow: https://gitlab.com/infuseai/primehub/-/blob/master/modules/charts/primehub/scripts/jupyterhub/config/jupyterhub_profiles.py#L474

I don't want to change jupyterhub_profile.py for this bug.